bentoml/yatai/client/bento_repository_api.py

```diff
@@ -52,13 +57,67 @@
     safe_retrieve,
     load_from_dir,
 )
+from bentoml.yatai.proto.yatai_service_pb2_grpc import YataiStub
 from bentoml.yatai.status import Status
 
 
 logger = logging.getLogger(__name__)
 yatai_proto = LazyLoader('yatai_proto', globals(), 'bentoml.yatai.proto')
 
 
+class BentoUploadStreamingRequests:
```

Comment:
Maybe `BentoUploadStreamRequests`? Why does upload behavior need to be modeled as a class?

Reply:
streaming grpc calls requires an iterator.  The `BentoUploadStreamRequests` class will be iterating the file and generate requests for the call.

Reply:
Got it. Not a big deal but perhaps we can do this inline with `yield`.

```python
while sent_chunk_count <  bundle_chunk_count:
    chunk = bundle.read(CHUNK_SIZE)
    yield UploadBentoRequest(chuck)
```